src/terms/xsd_type.rs

```diff
@@ -0,0 +1,71 @@
+// XSD Primitive Types that we're going to support
+// -----------------------------------------------
+// integer  - 32 bit - chosen as i32 is the default integer type in rust
+// boolean
+// date
+// dateTime
+// double   - 64 bit - chosen because f64 is the default floating-point type in rust
+// string
+// time
+
+use crate::terms::NamedNode;
+use chrono::{Date, DateTime, Utc};
```

Comment:
I'm getting a lot of warnings when testing locally about some of these `chrono` functions / structs being deprecated -- specifically `Date` and some of `Utc`'s associated functions.

Reply:
I updated my rust version and am still not seeing these warnings. Let's review tomorrow.

Reply:
I've updated my compiler version and am still not seeing warnings. Let's review tomorrow.